lib/components/dashboard/Buttons/Sun.tsx

```diff
@@ -6,7 +6,7 @@ import {cn} from '~/utils'
 export const Sun = ({className}: {className?: string}) => {
 	return (
 		<Button
-			onClick={() => signIn('github', {callbackUrl: '/dashboard'})}
```

Comment:
Why does this redirect to `/` after login now? Does the `/` path have its own redirect logic?

Reply:
We don't have a `/dashboard` route anymore, it's a dynamic route which means we check for session on the home page `/` and redirect to the relevant `/[team]`,